poetry.lock

```diff
@@ -71,7 +71,7 @@ python-versions = "*"
 
 [[package]]
 name = "certifi"
-version = "2020.11.8"
+version = "2020.12.5"
```

Comment:
I don't know why those changes happen to the lock file, any idea?

Reply:
I just created https://github.com/strawberry-graphql/strawberry/pull/627 . After that one is merged, this PR can be rebased from main branch.